src/main.rs

```diff
@@ -304,6 +566,8 @@ fn main() {
     signal_hook::flag::register(signal_hook::consts::SIGTERM, Arc::clone(&term)).unwrap();
     signal_hook::flag::register(signal_hook::consts::SIGINT, Arc::clone(&term)).unwrap();
 
+    let mut frame_acquire = !initial_config.is_audio_device().unwrap_or_default();
+
```

Comment:
If we're not acquiring frames in audio mode, should we even spin up the frame-socket thread?

Reply:
at the moment its got the logic for restart rp2040, so is always needed